library/find_unused_disk.py

```diff
@@ -106,19 +112,16 @@ def run_module():
         # If partition table exists but contains no partitions -> no partitions.
         no_partitions = not bool(ansible_facts['devices'][disk]['partitions'])
 
+        ansible_disk_size = ansible_facts['devices'][disk]['size'].lower().replace('gb', 'g').replace('mb', 'm')
+        disk_size = Size(ansible_disk_size)
+        min_disk_size = Size(module.params['disk_size'])
+
         if no_partitions and no_signature(run_command, '/dev/' + disk) and no_holders(disk) and can_open('/dev/' + disk):
-            result['disks'].append(disk)
+            if min_disk_size.bytes <= disk_size.bytes:
+                result['disks'].append(disk)
 
             if len(result['disks']) >= module.params['max_return']:
                 break
-        else:
-            disk_info = { 'name': disk,
-                          'partitions': no_partitions,
-                          'signature': no_signature(run_command, '/dev/' + disk),
-                          'holders': no_holders(disk),
-                          'can_open': can_open('/dev/' + disk)
-            }
-            result['used_disks'].append(disk_info)
```

Comment:
When was this added? Is it just some debug artifacts?

Reply:
I was using it purely for debugging purposes for the centos/rhel 6 VMs and was erased in the most current commit. 

I'll have to rebase all these commits soon.